devops/connectors/Dockerfile

```diff
@@ -1,14 +1,14 @@
 # Context - root of java project
 # Build Stage
-FROM cm/java-base:1.0.0 AS build
-COPY . /usr/src/project
-WORKDIR /usr/src/project
+FROM openjdk:8u131-jdk AS build
+COPY ./ /cm
+WORKDIR /cm
 RUN ./gradlew clean :connectors:installDist
 
 # Container with application
 FROM openjdk:8u131-jre-alpine
-WORKDIR /usr/src/connector
-COPY --from=build /usr/src/project/connectors/build/install/connectors .
+COPY --from=build /cm/connectors/build/install/connectors /cm/connectors/dist
+WORKDIR /cm/connectors/dist
 CMD ["./bin/connectors"]
```

Comment:
So, i should move binaries not to /cm/connectors/dist but for /cyberapp/cm-connectors?
Could you please, provide example of how i should using VOLUME?